mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -30,8 +30,20 @@
   public static final String DIVIDER = ">>>";
   public static final String SERVICE_NAME = MortarScope.class.getName();
 
+  /**
+   * Retrieves a MortarScope from the given context. If none is found, retrieves a MortarScope from
+   * the application context.
+   *
+   * @return null if no scope is found in either the given context or the application context
+   */
   public static MortarScope getScope(Context context) {
-    return (MortarScope) context.getSystemService(MortarScope.SERVICE_NAME);
+    //noinspection ResourceType
+    Object scope = context.getSystemService(SERVICE_NAME);
+    if (scope == null) {
+      //noinspection ResourceType
+      scope = context.getApplicationContext().getSystemService(SERVICE_NAME);
```

Comment:
`ContextThemeWrapper` extends `ContextWrapper` and `getApplicationContext()` is implemented in `ContextWrapper`:

```java
    @Override
    public Context getApplicationContext() {
        return mBase.getApplicationContext();
    }
```

`ContextThemeWrapper` doesn't implement it's own `attach()` method:

```
    @Override protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(newBase);
    }
```

I can't research this because I have no idea exactly what the problem was and how this is supposed to fix it. But from the looks of it, this change does nothing.